minicluster/src/main/java/alluxio/multi/process/MultiProcessCluster.java

```diff
@@ -162,15 +162,23 @@ public synchronized void start() throws Exception {
     mState = State.STARTED;
 
     mMasterAddresses = generateMasterAddresses(mNumMasters);
+    MasterNetAddress masterAddress = mMasterAddresses.get(0);
     LOG.info("Master addresses: {}", mMasterAddresses);
     switch (mDeployMode) {
-      case NON_HA:
-        MasterNetAddress masterAddress = mMasterAddresses.get(0);
+      case UFS_NON_HA:
         mProperties.put(PropertyKey.MASTER_JOURNAL_TYPE, JournalType.UFS.toString());
         mProperties.put(PropertyKey.MASTER_HOSTNAME, masterAddress.getHostname());
         mProperties.put(PropertyKey.MASTER_RPC_PORT, Integer.toString(masterAddress.getRpcPort()));
         mProperties.put(PropertyKey.MASTER_WEB_PORT, Integer.toString(masterAddress.getWebPort()));
         break;
+      case EMBEDDED_NON_HA:
```

Comment:
Why do we need a separate `EMBEDDED_NON_HA` from `EMBEDDED_HA`? It looks like they result in the same configuration.

Reply:
The reason why I use `EMBEDDED_NON_HA` and `EMBEDDED_HA` before is because I found out the `EMBEDDED_HA` configuration with embedded journal addresses and master rpc addresses do not work. This is because our workers still use `master.rpc.port` when building the master inquire client and cannot connect to the leading master. This bug has been fixed in the PR.

Reply:
Can you create an issue to track this bug?

Reply:
This bug is already fixed in this PR, but I will open an issue to describe this bug and point the this PR.